src/Avalonia.Themes.Fluent/Controls/MenuFlyoutPresenter.xaml

```diff
@@ -32,4 +32,39 @@
       </ControlTemplate>
     </Setter>
   </ControlTheme>
+  <ControlTheme x:Key="MobileHorizontalMenuFlyoutPresenter" TargetType="MenuFlyoutPresenter">
```

Comment:
I don't think it's necessary to implement my idea in this PR, but in general, we can have TextBox.SelectionFlyout property, keeping it consistent with other Flyout props and UWP.
https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textbox.selectionflyout?view=winrt-22621
And TextCommandBarFlyout class that has a default selection 
https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textcommandbarflyout?view=winrt-22621

Reply:
In general, UWP has ContextFlyout and SelectionFlyout independent:
![image](https://github.com/AvaloniaUI/Avalonia/assets/3163374/a92c5566-2bd1-4acb-a9ec-38d3367db1bd)

![image](https://github.com/AvaloniaUI/Avalonia/assets/3163374/3907ccb9-855b-47f6-a9ff-72b0c8e83fd0)

Reply:
For now, let's keep it simpler. It doesn't seem to be a big breaking change later.

Reply:
This sounds like a great idea. There may be some issues with how to trigger both separately, as the concept of different menus for different intents on edit fields doesn't exist on mobile. Maybe Hold to trigger the default menu, and tapping the handles or ending a selection gesture to trigger the selection menu.